src/cli/cmds/sync.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import path from 'path';
+import {program} from 'commander';
+import {
+  getRechunkConfig,
+  getIOSPath,
+  getAndroidPath,
+  doesKeywordExist,
+  updateFile,
+  LOGO,
+} from '../lib';
+
+/**
+ * Defines a command for the "sync" operation using the "commander" library.
+ *
+ * @example
+ * ```bash
+ * yarn rechunk sync
+ * ```
+ *
+ * @remarks
+ * This command is part of a larger program defined using the "commander" library.
+ *
+ * @see {@link https://www.npmjs.com/package/commander | commander} - Command-line framework for Node.js.
+ */
+program
+  .command('sync')
+  .description('sync keys from the ReChunk configuration with native files')
+  .action(async () => {
+    console.log();
+    console.log(LOGO);
+
+    const rc = getRechunkConfig();
+
+    const iosPath = getIOSPath();
+    const infoPlistPath = path.resolve(iosPath, 'example', 'Info.plist');
```

Comment:
@crherman7 could you please tell me how to correctly get this value dynamically?

Reply:
We will likely want to use https://github.com/isaacs/node-glob to find the plist file for us with the use of globs

Reply:
A good example use case that can draw some inspiration is from `react-native-navigation` actually hahah - https://github.com/wix/react-native-navigation/blob/master/autolink/postlink/path.js.

Reply:
@crherman7 great package, thanks for the tip 👍